packages/logs/src/domain/assembly.spec.ts

```diff
@@ -270,6 +277,52 @@ describe('startLogsAssembly', () => {
   })
 })
 
+describe('user management', () => {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏As you pointed out, we could have conflict between user context and global context, we should have a test to guarantee the behavior on that front